bundles/org.openhab.binding.juicenet/src/main/java/org/openhab/binding/juicenet/internal/handler/JuiceNetDeviceHandler.java

```diff
@@ -255,6 +255,13 @@ public void handleCommand(ChannelUID channelUID, Command command) {
 
                     break;
                 }
+                case CHANNEL_OVERRIDE: {
+                    String state = ((OnOffType) command).toString();
+                    if (state.equals("OFF")) {
+                        api.setOverride(Objects.requireNonNull(token), 0, ((long) 0), 0);
+                    }
+
```

Comment:
```suggestion

```

Reply:
I think adding the break; makes this OBE

Reply:
Yes